blogs_notebooks/titanic_data_science_solutions_blog_1.ipynb

```diff
@@ -0,0 +1,492 @@
+{
```

Comment:
<p>it would be nice to understand the why.  it could be like:</p><p> </p><p>"Let's see how this works by find out the average age and fare of the passengers who embarked.  We can also filter out older passengers and those who paid a lot of money"</p><p> </p><p>actually, why did you filter those out?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai-community/showcase/pull/56/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>